src/odemis/gui/model/__init__.py

```diff
@@ -338,8 +338,10 @@ def __init__(self, microscope):
             required_roles = []
             if self.role in ("secom", "delphi", "enzel"):
                 required_roles += ["e-beam", "light", "stage", "focus"]
-                if self.role == "secom":
+                if self.role in ("secom", "enzel"):
```

Comment:
This mean we can no longer use an enzel outside of the microscope isn't it?
On second thought I think the application/installation engineers still do this quite often. Maybe there should be some option to still use it (like role:enzel_door_only)

Reply:
No. This code doesn't really change the behaviour of the GUI, it just makes it "nicer" to the user. Currently, if starting a microscope file with role "enzel" missing some component, the GUI will just fail to start with some weird error like "object of NoneType has no attribute beamShift". Now, it will also fail, but saying "Microscope is missing ion-beam". 

There are two ways to start a system with less hardware:
* simulate some of the components (that's often the most useful way)
* use a different microscope type (eg, optical)

Reply:
Ok sounds great